pkgs/tools/text/ripgrep/default.nix

```diff
@@ -1,19 +1,24 @@
-{ stdenv, fetchFromGitHub, rustPlatform, asciidoc, docbook_xsl, libxslt  }:
+{ stdenv, fetchFromGitHub, rustPlatform, asciidoc, docbook_xsl, libxslt
+, withPCRE2 ? false, pcre2 ? null
```

Comment:
Maybe `withPcre` or `withpcre`?

Reply:
I think it's fine as is. Camel casing looks weird for abbreviations anyway.